### PR TITLE
Update JoinWaitlist.js

### PR DESCRIPTION
**Pull request by StewPoll**
This PR updates the "Join waitlist" section to play the notification alarm when you join the waitlist, so that when you get an invite the alarm can play as well. 

Haven't got local development setup yet, but feel that this should be good to go.

### CODE REVIEW

**Comment by samuelgrant** [comment]
Probably don't want to play sound when someone joins up. Is there a way we could load the file in within the notification widget during x-up but without playing sound?

**Comment by StewPoll** [comment]
AFAIK, no.

Chrome 46 onwards prevents Tabs playing audio in the background, which functionally means it's fine in the background, if its played in focus first. 

So loading the file isn't going to since the problem of it being in the background when it goes to play.

**Comment by samuelgrant** [comment]
I was wondering if the option might be to play it muted.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: frontend/src/Pages/Waitlist/JoinWaitlist.js [hunk 1]
```diff
@@ -123,6 +131,9 @@ const JoinWaitlist = ({ hasFits }) => {
           });
           reset();
           setOpen(false);
+          if (settings.audio_alarm) {
+            new Audio(notificationAlarm).play()
```
Comment (StewPoll) [inline frontend/src/Pages/Waitlist/JoinWaitlist.js]:
```suggestion
            let alert = Audio(notificationAlarm)
            alert.volume = 0.1
            alert.play()
```

**Comment by StewPoll** [comment]
@samuelgrant I don't think doing it muted would satisfy chrome's weirdness. Play it quitely instead.

**Pull request merged**